### PR TITLE
add description to stack collapse tests

### PR DESCRIPTION
**Pull request by andjo403**

### CODE REVIEW

File: tools_lib/src/stack_collapse.rs [hunk 1]
```diff
@@ -56,116 +56,170 @@ mod test {
 
     #[test]
     fn basic_test() {
+        //                                         <--e1-->
```
Comment (michaelwoerister) [inline tools_lib/src/stack_collapse.rs]: Should this be `e3`?

File: tools_lib/src/stack_collapse.rs [hunk 2]
```diff
@@ -56,116 +56,170 @@ mod test {
...
+        //                 <--e1-->        <----------e2---------->
+        //              T2 1       2       3       4       5       6
+        // sample interval |   |   |   |   |   |   |   |   |   |   |
+        // stacks count:
+        // rustc                       1   2
+        // rustc;e1            1   2
+        // rustc;e2                            1   2           3   4
+        // rustc;e2;e3                                 1   2
```
Comment (andjo403) [inline tools_lib/src/stack_collapse.rs]: think this shall be e1 want to use the same event to show that it is on the stack here
```suggestion
        // rustc;e2;e1                                 1   2
```

File: tools_lib/src/stack_collapse.rs (hunk 1 above)
Comment (andjo403) [reply tools_lib/src/stack_collapse.rs]: see my comment below

File: tools_lib/src/stack_collapse.rs (hunk 2 above)
Comment (michaelwoerister) [reply tools_lib/src/stack_collapse.rs]: :+1:

**Comment by andjo403** [comment]
fixed the fault

**Pull request merged**